### ISSUE TITLE
Feature request: Allow to disable Webkit scrollbar styling

### ISSUE THREAD

**Issue by jothepro** [doxygen/doxygen#9549]
In https://github.com/doxygen/doxygen/commit/260f2fadf8b780d824cba2e490d119e759ada101 custom styling for scrollbars was introduced.

I'd prefer to keep the default browser scrollbar in some places, though.
Unfortunately it is [not possible to deactivate the scrollbar customization by adding custom CSS on top of it](https://superuser.com/questions/380629/is-there-a-way-to-disable-custom-webkit-scrollbars).

This is why I need to be able to disable  the scrollbar customization in the Doxyfile, so it is not added to `doxygen.css` in the first place.

E.g. `DISABLE_SCROLLBAR_STYLING = YES`.

What's your take on that? As an alternative I could entirely replace `doxygen.css` with my own stylesheet, but AFAIK this is not recommended and I'd like to avoid it.

**Comment by doxygen** [comment]
@jothepro This is indeed an unfortunate limitation I did not foresee. 

Actually seems like a bug introduced in recent versions of chromium.

In the past:

```css
::-webkit-scrollbar {
    all:unset;
}
```

apparently worked to reset the style.

See here for a potential workaround: https://stackoverflow.com/a/72511916/784672
If it doesn't work, I'll see if an option can be added.

**Comment by jothepro** [comment]
Yep, it seems like this doesn't work:

<img width="300" alt="Bildschirmfoto 2022-09-01 um 10 22 16" src="https://user-images.githubusercontent.com/21294002/187867671-25424b6e-1914-4271-b2bc-767b74760933.png">

Taking this further and unsetting all the styles that have been applied to the scrollbar will lead to an invisible/hidden scrollbar.

The closest I can get to the original look on Windows is this:




<img width="300" alt="Bildschirmfoto 2022-09-01 um 10 20 50" src="https://user-images.githubusercontent.com/21294002/187867373-c57008ba-2a24-4201-990d-b8f8162c85ee.png">

<details>
  <summary>Applied CSS (Source: https://superuser.com/a/1685163)</summary>
  <p>

```css
*::-webkit-scrollbar {
    all: initial !important;
    width: 15px !important;
    background: #d8d8d8 !important;
}
*::-webkit-scrollbar-thumb {
    all: initial !important;
    background: #7c7c7c !important;
}

*::-webkit-scrollbar-button { all: initial !important; }
*::-webkit-scrollbar-track { all: initial !important; }
*::-webkit-scrollbar-track-piece { all: initial !important; }
*::-webkit-scrollbar-corner { all: initial !important; }
*::-webkit-resizer { all: initial !important; }
```

</p>
</details>


But this won't work on macOS and on Android where the default scrollbars look and behave very different. 😞

**Comment by doxygen** [comment]
@jothepro To avoid introducing yet another option, I'm thinking about adding the `::-webkit-scrollbar` stuff to `doxygen.css` only when no `HTML_EXTRA_STYLESHEET` has been specified. With the idea that you can always add back the scrollbar styling in the extra stylesheet if you need it. What do you think?

**Comment by jothepro** [comment]
This would also solve my problem. 👍

**Comment by doxygen** [comment]
@jothepro Please verify if the referenced commit indeed solves the problem. Please do not close the issue, it will be done automatically when a new release is made available.

**Comment by jothepro** [comment]
I can confirm that this solves my problem! Thanks a lot! 😊

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.6.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
